fe/src/main/java/org/apache/doris/transaction/TransactionCommitFailedException.java

```diff
@@ -22,6 +22,8 @@
     
     private static final long serialVersionUID = -2528170792631761535L;
 
+    public static final String NO_DATA_TO_LOAD_MSG = "all partitions have no load data";
```

Comment:
You don't need to judge NO_DATA_TO_LOAD_MSG. In my option, all of exceptions need to be catch and record in old user pattern

Reply:
OK